Editor/Editors/CinemachineScreenComposerGuidesGlobalEditable.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using UnityEditor;
+
+namespace Cinemachine.Editor
+{
+    [InitializeOnLoad]
+    class CinemachineScreenComposerGuidesGlobalEditable // why not static? (same Q for CinemachineStoryboardMute)
```

Comment:
Should I move this in Editor/Editors/CinemachineScreenComposerGuides.cs?

Reply:
yes please.
And yes, I suppose it could be a static class.  Can you find out if it makes a difference, and explain to me what that difference is?

Reply:
**Reason for Static class**
- "A static class is basically the same as a non-static class, but there is one difference: a static class cannot be instantiated." [1]
- Static class is more restrictive than a normal class. Both classes (CMStoryboardMute and CMScreenComposerGuidesEditable) fit into this more restrictive rule set. 

The more restrictions are:
1. contains only static members,
2. cannot be instantiated,
3. is sealed, and
4. cannot contain Instance Constructors. [1]

**Reason for InitializeOnLoad attribute and static constructor:**
-  "A static constructor is used to initialize any static data, or to perform a particular action that needs to be performed once only. It is called automatically before the first instance is created or any static members are referenced." [1]
- Static constructors with InitializeOnLoad attribute are called when scripts in the project are recompiled. [2]

[1] https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/static-classes-and-static-class-members
[2] https://docs.unity3d.com/ScriptReference/InitializeOnLoadAttribute.html